Clinical-Genomics/scout
# Fix #3022 - add melter toggle

This PR adds a functionality or fixes a bug.

![Screenshot 2021-12-03 at 16 26 26](https://user-images.githubusercontent.com/758570/144628244-62e5ba9e-f8fb-485d-aa51-13a982c6f285.png)

![Screenshot 2021-12-03 at 17 14 09](https://user-images.githubusercontent.com/758570/144635455-cb62ef28-2a1e-4870-8a00-33245a1f5060.png)

![Screenshot 2021-12-03 at 17 31 18](https://user-images.githubusercontent.com/758570/144638242-c70e6c05-087e-447c-b106-e4c79c827742.png)

![Screenshot 2021-12-03 at 17 39 10](https://user-images.githubusercontent.com/758570/144639520-a2b6db36-9b01-4a1b-a94f-01ff61525be7.png)

**How to test**:
1. toggle rerun monitoring on and off for cases
2. note that this should be logged as events for the case
3. when rerun monitoring is enabled for a case, it should be exported ok using `scout export cases --rerun-monitor`

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
